deepchem/feat/molecule_featurizers/mat_featurizer.py

```diff
@@ -0,0 +1,67 @@
+from deepchem.feat.base_classes import MolecularFeaturizer
+from deepchem.utils.molecule_feature_utils import one_hot_encode
+try:
+  import logging
+  import numpy as np
+  import pandas as pd
+  import torch
+  from rdkit import Chem
+  from rdkit.Chem import AllChem
+  from rdkit.Chem import MolFromSmiles
+  from sklearn.metrics import pairwise_distances
+  from torch.utils import datasets
+except:
+  raise ImportError("Required Modules not found.")
+
+class MATFeaturizer(MolecularFeaturizer):
+
+  def __init__(
+      self,
+      add_dummy_node=True,
+      one_hot_formal_charge=True,
+  ):
+    self.mol = mol
+    self.add_dummy_node = add_dummy_node
+    self.one_hot_formal_charge = one_hot_formal_charge
+
+  def atom_features(self, atom, one_hot_formal_charge=True):
+    attrib = []
+    attrib += one_hot_encode(atom.getAtomicNum(),
```

Comment:
`atom.GetAtomicNumber` I think

Reply:
I just checked the Rdkit docs, it says getAtomicNum [here](https://www.rdkit.org/docs/source/rdkit.Chem.rdchem.html)

Reply:
It’s GetAtomicNum instead of getAtomicNum

Reply:
Good catch! Thank you, I have pushed the new change now.